SpatialGDK/Source/SpatialGDKTests/Private/LocalDeploymentManager/LocalDeploymentManagerTest.cpp

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "LocalDeploymentManager.h"
+#include "SpatialGDKDefaultLaunchConfigGenerator.h"
+#include "SpatialGDKDefaultWorkerJsonGenerator.h"
+#include "SpatialGDKEditorSettings.h"
+
+#include "CoreMinimal.h"
+
+#define LOCALDEPLOYMENT_TEST(TestName) \
+	TEST(Services, LocalDeployment, TestName)
+
+namespace
+{
+	static const double MAX_WAIT_TIME_FOR_LOCAL_DEPLOYMENT_OPERATION = 10.0;
+	enum class EDeploymentState { IsRunning, IsNotRunning };
+
+	FLocalDeploymentManager* GetLocalDeploymentManager()
+	{
+		FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
```

Comment:
Can we make the module and the manager const please?

Reply:
Don't think you can when using this method.

Reply:
No.

Reply:
We use it's non-const methods.